server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -244,21 +245,6 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
 
     void get_cpuload(float& peak, float& average) const { cpu_time_accumulator.get(peak, average); }
 
-    std::pair<std::string, std::string> default_device_names() {
-        const PaDeviceIndex default_input = Pa_GetDefaultInputDevice();
-        const PaDeviceIndex default_output = Pa_GetDefaultOutputDevice();
-
-        std::cout << default_input << " " << default_output;
-
-        return std::make_pair(device_name(default_input), device_name(default_output));
-    }
-
-private:
```

Comment:
just noticed - is it an issue I removed this?

Reply:
yes, it'll make perform public, please add it back :)